### DISCUSSION TITLE
`git pull --rebase=interactive` support

### DISCUSSION THREAD

**Discussion by sin-ack** [magit/magit#5224] [Ideas]
Hey, thanks so much for working on Magit! I wanted to ask whether Magit is planning to, or would like to support `git pull --rebase=interactive`? I use it at work a lot for rebasing "stacked PRs" (where one PR builds on top of another, and I rebase when the dependency PR is merged to master), and I currently need to fall back to the CLI :(

**Comment by kyleam** [comment]
> and I currently need to fall back to the CLI :(

Couldn't you set `pull.rebase`  (or `branch.<name>.rebase`) to `interactive` ?

gh-4863 has some discussion about updating `--rebase` in the `magit-pull` transient to read a value.

**Reply by sin-ack** [reply]
In most cases it's a one-off thing, and I work with many different branches so setting it manually for each branch is cumbersome. Ideally I could do something like `F -r i u` and have it work exactly like `r i`. I want to see whether I can prepare a patch for it myself.

**Comment by tarsius** [comment]
I've added support for this in d79bba7665c4f878e65d17957649d2b2c05f5e27.
